### DISCUSSION TITLE
Routing support?

### DISCUSSION THREAD

**Discussion by ksckaan1** [cogentcore/core#1263] [Q&A] [answered]
Does cogent core support routing internally? If no, is there such a plan?

**Comment by kkoreilly** [comment, marked as answer]
Do you mean having multiple pages within one app? If so, you can look at [pages](https://www.cogentcore.org/core/widgets/other/pages). You can also see https://github.com/cogentcore/core/issues/1108#issuecomment-2294994762 for another option. If that is not what you are asking, please clarify. Thank you!

**Reply by ksckaan1** [reply]
I mean run full dialog. Thank you for your response

**Reply by kkoreilly** [reply]
@ksckaan1 You may also be interested in our new `core.Pages` feature (see the [sign in tutorial](https://www.cogentcore.org/core/tutorials/sign-in), #1293, #1289).

**Reply by ksckaan1** [reply]
This is exact thing that i look. Thank you!
